Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -526,22 +525,19 @@ public class BrowserViewController: UIViewController, BrowserViewControllerDeleg
     updateWidgetFavoritesData()
     
     Task { @MainActor in
+      self.setupTabs()
```

Comment:
Since we are calling this first and on the main actor I think you can just call it outside the task now, no?

Reply:
If I do that it is crashing at https://github.com/brave/brave-ios/blob/113cc3da2230fbffce13a7ad2884485fac389453/Client/Frontend/Browser/BrowserViewController.swift#L1425

Reply:
Maybe something wrong with how we load those views. Im ok with you keeping it this way then and we can fix this in another PR